lib/addons/p5.dom.js

```diff
@@ -675,6 +675,84 @@
   };
 
   /**
+   * Translates an element relative to (0, 0, 0) of the window through
+   * css transforms. 
+   * @method  translate
+   * @param  {Number} x x-position relative to upper left of window
+   * @param  {Number} y y-position relative to upper left of window
+   * @param  {Number} z z-position
+   * @return {p5.Element}
+   * @example
+   * <div><code class='norender'>
+   * function setup() {
+   *   var cnv = createCanvas(100,100);
+   *   //translates canvas 50px to the right and 100px
+   *   //below upper left corner of the window
+   *   cnv.translate(50,100);
+   * }
+   * </code></div>
+   */
+  p5.Element.prototype.translate = function(x, y, z){
+    this.elt.style.position = 'absolute';
+    var regEx = new RegExp('translate');
+    var substr = '';
+    while (regEx.test(this.elt.style.transform)){
```

Comment:
rather than doing this manually, I think you could just make a generic regex that searchs for the pattern translate......) and replace any of those with "".